app/celery/tasks.py

```diff
@@ -82,6 +83,7 @@
 def process_job(job_id, sender_id=None):
```

Comment:
Did not remove the kwarg just yet, to avoid breaking things when deploying. But this can be removed later on without problems